ui/src/components/toast-error.tsx

```diff
@@ -17,14 +17,25 @@
  * License-Filename: LICENSE
  */
 
+import { ApiError } from '@/api/requests';
```

Comment:
Please add a bit more background information to the commit message, maybe also referring to 0fd185826671f3c6c4bda47bcefb00d44c6936a6 which introduced those `any` casts just recently.